src/Execution/ExecutionStrategy.php

```diff
@@ -172,27 +172,31 @@ protected function executeFields(
      * @param Schema     $schema
      * @param ObjectType $parentType
      * @param string     $fieldName
-     * @return \Digia\GraphQL\Type\Definition\Field|null
+     * @return Field|null
      * @throws \Exception
      * @throws \TypeError
      */
     public function getFieldDefinition(Schema $schema, ObjectType $parentType, string $fieldName)
     {
-        if ($fieldName === SchemaMetaFieldDefinition()->getName() && $schema->getQuery() === $parentType) {
-            return SchemaMetaFieldDefinition();
+        $schemaMetaFieldDifinition   = SchemaMetaFieldDefinition();
```

Comment:
I would move these just before the associated `if` -statement because we're using early return.